Minecraft.Client/Blocks/Stone.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Decent.Minecraft.Client.Blocks
+{
+    public class Stone : Block
+    {
+        public Stone(StoneVariants variant = StoneVariants.Stone) : base(BlockType.Stone)
+        {
+            Variant = variant;
+        }
+
+        public StoneVariants Variant { get; }
```

Comment:
Sure! I just went with the naming from http://minecraft.gamepedia.com/Stone, but by all means; refactor if it makes it better :)